sdk/core/azure-core/inc/azure/core/http/policy.hpp

```diff
@@ -359,18 +360,42 @@ namespace Azure { namespace Core { namespace Http {
         NextHttpPolicy policy) const override;
   };
 
+  /**
+   * @brief Options for Azure::Core::Http::LoggingPolicy.
+   */
+  struct LoggingPolicyOptions
+  {
+    /**
+     * @brief HTTP query parameters that are allowed to be logged.
+     * @remark URL-encoded parameter names are expected.
+     */
+    std::set<std::string> AllowedHttpQueryParameters;
+
+    /**
+     * @brief HTTP request headers that are allowed to be logged.
+     */
+    Azure::Core::CaseInsensitiveSet AllowedHttpRequestHeaders;
+
+    /**
+     * @brief HTTP response headers that are allowed to be logged.
+     */
+    Azure::Core::CaseInsensitiveSet AllowedHttpResponseHeaders;
+  };
+
   /**
    * @brief Logs every HTTP request.
    *
-   * @details Logs every HTTP request, response, or retry attempt.
-   * @remark See #logging.hpp
+   * @details Logs every HTTP request and response.
+   * @remark See Azure::Core::Logger.
    */
   class LoggingPolicy : public HttpPolicy {
```

Comment:
Renaem to LogPolicy

Reply:
Why do we prefer the name `LogPolicy` over `LoggingPolicy`?

.Net: [LoggingPolicy](https://github.com/Azure/azure-sdk-for-net/blob/49d9b2c3be3d48fc6d8f7044ae74c20c2cfc34c0/sdk/core/Azure.Core/src/Pipeline/Internal/LoggingPolicy.cs#L16)
Java: [HttpLoggingPolicy](https://github.com/Azure/azure-sdk-for-java/blob/d7bcc73f3866dbfb007a6ae8512b1fb01ac58ada/sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpLoggingPolicy.java#L39)
Python: [HttpLoggingPolicy](https://github.com/Azure/azure-sdk-for-python/blob/4f1c4a7bff4cf9a50f5d882e1f8adf8663adcadb/sdk/core/azure-core/azure/core/pipeline/policies/_universal.py#L347)
JS: [LogPolicy](https://github.com/Azure/azure-sdk-for-js/blob/50c5a5447454c2ae1d2b4fa5b02470671f41dd77/sdk/core/core-http/src/policies/logPolicy.ts#L45)